bot/util.go

```diff
@@ -447,13 +447,20 @@ func CheckDiscordErrRetry(err error) bool {
 	return true
 }
 
-func IsNormalUserMessage(msg *discordgo.Message) bool {
+var allowedMsgTypes = []int{int(discordgo.MessageTypeDefault), int(discordgo.MessageTypeReply), int(discordgo.MessageTypeThreadStarterMessage)}
```

Comment:
What do you think about inlining the check and instead writing:
```go
switch msg.Type {
case discordgo.MessageTypeDefault, discordgo.MessageTypeReply, discordgo.MessageTypeThreadStarterMessage:
	return true
default:
	return false
}
```
This avoids the cast, is a little shorter, and is more performant (though that isn't the main reasoning for this suggestion, to be clear.) We also use this pattern in a couple other places.

Reply:
Oo I like it, didn’t cross my mind.